worker/buildbot_worker/test/util/misc.py

```diff
@@ -23,14 +23,16 @@
 except ImportError:
     # Python 3
     import builtins
-import cStringIO
 import errno
 import os
 import re
 import shutil
 import sys
+from io import BytesIO
```

Comment:
I'd say its ok. the from are placed after the imports within the stdlib import group